client/my-sites/site-settings/site-owner-transfer/start-site-owner-transfer.tsx

```diff
@@ -21,6 +22,7 @@ type Props = {
 	selectedSiteSlug: string | null;
 	siteOwner: User;
 	customDomains: ResponseDomain[];
+	isAtomicSite: any;
```

Comment:
Could we follow the type returned by `isSiteAutomatedTransfer` and define it as `boolean | null`?